tools/src/test/resources/dev/run-qualification-tests.sh

```diff
@@ -0,0 +1,395 @@
+   log_msg "                                  pass it as --cp=\$SPARK_HOME/jars/*:\$CLASS_PATH"
+   log_msg "  --out-dir=<arg>               - Output directory passed to the qualification tool runs."
+   log_msg "                                  By default is is set to test/resources/dev/qualification-output"
+   log_msg "                                  The directory has two subdirectories: csv/ which has the CSV files; and"
+   log_msg "                                  runs-output/ which has the actual output of the tool."
+   log_msg "                                  target directory of tools."
+   log_msg "  --qual-expectations-dir=<arg> - Path of the reference CSV files."
+   log_msg "                                  By default is is set to test/resources/QualificationExpectations"
+   log_msg "  --qual-events-dir=<arg>       - Path of the Spark events logs used as input for the unit tests"
+   log_msg "                                  By default is is set to test/resources/spark-events-qualification"
+   log_msg "  --prof-log-dir=<arg>          - Some the unit tests reads files located in another directory."
+   log_msg "                                  By default is is set to test/resources/spark-events-profiling"
+   log_msg "  --heap=<arg>                  - optional heap size. Default is 10g."
+   log_msg "  --help|-h                     - Shows Help."
+   log_msg
+   log_msg "Example Usage:"
+   log_msg "  run-qualification-tests.sh --cp=\$CLASS_PATH --heap=5g"
+   log_msg "  This is equivalent to:"
+   log_msg "    java -Xmx5g \\"
+   log_msg "         -cp rapids-4-spark-tools_2.12-<version>-SNAPSHOT.jar:\$CLASS_PATH \\"
+   log_msg "         com.nvidia.spark.rapids.tool.qualification.QualificationMain \\"
+   log_msg "         --no-html-report --output-directory file:qualification-output/csv \\"
+   log_msg "         \$LOGFILES"
+   log_msg
+   log_msg "How to Add New Test:"
+   log_msg " 1- add a dummy csv file in qual-expectations-dir <new_unit_test.csv>"
+   log_msg " 2- add the eventlog into the prof-log-dir <new_unit_log>"
+   log_msg " 3- update the definition of the hash in define_qualification_tests_map() to map between"
+   log_msg "    the expected file and the unit test name."
+   log_msg " 4- run teh script. It is expected that the script fails because the dummy csv is incorrect."
+   log_msg " 5- copy the content of the generated output dev/qualification-output/csv/new_unit_test.csv"
+   log_msg "    into the expectation folders."
+   log_msg
+}
+
+log_error()
+{
+  log_msg "${RED}$*"
+}
+
+log_result()
+{
+  log_msg "${GREEN}$*"
+}
+
+log_info() {
+  log_msg "${YELLOW}$*"
+}
+
+log_msg()
+{
+  echo -e "$* $ENDCOLOR"
+}
+
+print_banner()
+{
+  printf '%80s\n' | tr ' ' -
+  log_info "$*"
+  printf '%80s\n' | tr ' ' -
+}
+
+set_rapids_jars_from_work_dir()
+{
+  rapids_tools_jar_file=( "$( find "${MODULE_PATH}" -type f \( -iname "*.jar" ! -iname "*tests.jar" ! -iname "original-rapids-4*.jar" \) )" )
```

Comment:
seems like it would be safer to find rapids-4-spark-tools_*.jar

Reply:
done!